taverna-cwl-activity-ui/pom.xml

```diff
@@ -112,6 +112,12 @@
 			<artifactId>jackson-databind</artifactId>
 			<version>${jackson.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
SnakeYAML [is Apache License 2.0](https://bitbucket.org/asomov/snakeyaml/src/70abb5efa4c0ec0dd3680383c9fabb5cd3939c80/LICENSE.txt?at=v1.17&fileviewer=file-view-default) - so OK to use. (Why not `1.18`?)